go.sum

```diff
@@ -89,6 +89,20 @@ github.com/ProtonMail/gopenpgp/v2 v2.7.4 h1:Vz/8+HViFFnf2A6XX8JOvZMrA6F5puwNvvF2
 github.com/ProtonMail/gopenpgp/v2 v2.7.4/go.mod h1:IhkNEDaxec6NyzSI0PlxapinnwPVIESk8/76da3Ct3g=
 github.com/PuerkitoBio/goquery v1.8.1 h1:uQxhNlArOIdbrH1tr0UXwdVFgDcZDrZVdcpygAcwmWM=
 github.com/PuerkitoBio/goquery v1.8.1/go.mod h1:Q8ICL1kNUJ2sXGoAhPGUdYDJvgQgHzJsnnd3H7Ho5jQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505204940-afd5267fe57f h1:719s3rc1mxCQfFn5B6l40lwGaewlA2qFvhJ8VzHXVK4=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505204940-afd5267fe57f/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505214629-764d0ec1512a h1:GY7UVDWm9vzv50icXw9MoLBOVOhaJKZJGKW4cQZDUdQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505214629-764d0ec1512a/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505215717-25bd7e9f0180 h1:d7Jl3LJANlw7rviAECdEt5NrfaY89ziPjuacL2TGKQ8=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505215717-25bd7e9f0180/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505220824-7a00a541234d h1:9T+fVJ6NqsX9c7fPuuIC3qV9xXX4R3YmIJyofqeFSBE=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505220824-7a00a541234d/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505223234-6fd1f77d7d00 h1:sfT8QZF4yMUUwmgIfbiEGzmuS/SHvMc+tvRWasujf40=
+github.com/StarHack/go-internxt-drive v0.0.0-20250505223234-6fd1f77d7d00/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250506081342-16852f088f5f h1:PJBZVoUwqPQZoV12s/TdMoRcyRfaCJwTbViyKS9Tgxs=
+github.com/StarHack/go-internxt-drive v0.0.0-20250506081342-16852f088f5f/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
+github.com/StarHack/go-internxt-drive v0.0.0-20250506081634-548256aeb125 h1:g9IlQWO9WCoy8DbJaee6hzR9ajdKVpTpE9vFOOWkk08=
+github.com/StarHack/go-internxt-drive v0.0.0-20250506081634-548256aeb125/go.mod h1:YmT8WhRhk9JTWrGJyuXsZdRU+poYZZIND5qXL7FZ2lQ=
```

Comment:
Don't think all of these are needed, and you can run `go mod tidy` to clean up a bit?